fsharp-backend/src/LibExecution/StdLib/LibList.fs

```diff
@@ -793,19 +801,24 @@ let fns : List<BuiltInFn> =
 //     ; sqlSpec = NotYetImplementedTODO
 //     ; previewable = Pure
 //     ; deprecated = NotDeprecated }
-//   ; { name = fn "List" "take" 0
-//     ; parameters = [Param.make "list" TList ""; Param.make "count" TInt ""]
-//     ; returnType = TList
-//     ; description = "Drops all but the first `count` values from `list`."
-//     ; fn =
-//           (function
-//           | _, [DList l; DInt c] ->
-//               DList (List.take l (Dint.to_int_exn c))
-//           | _ ->
-//               incorrectArgs ())
-//     ; sqlSpec = NotYetImplementedTODO
-//     ; previewable = Pure
-//     ; deprecated = NotDeprecated }
+    { name = fn "List" "take" 0
+      parameters = [Param.make "list" (TList varA) ""; Param.make "count" TInt ""]
+      returnType = TList varA
+      description = "Drops all but the first `count` values from `list`."
+      fn =
+        (function
+        | _, [DList l; DInt c] ->
+              if int c > 0 then
```

Comment:
This fn would also be clearer IMO if we put the error conditions first.

Reply:
I agree too. Thank you.